### ISSUE TITLE
There was an 'Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException' error when opening save files in 1.13.3

### ISSUE THREAD

**Issue by Pd69bq** [goatfungus/NMSSaveEditor#787]
the following error occurs when opening up PS4 save files in version 1.13.3, don't have PC saves and the game, cannot try. 
the``Save Name`` and ``Description`` failed to load and still showing "unknown", only ``Exosuit`` tab works, other tabs either failed to load or grayed out.

```
[DEBUG] Path not found: PlayerStateData.DifficultyState.Settings.InventoryStackLimits.InventoryStackLimitsDifficulty
[DEBUG] Path not found: StackSizeGroup.InventoryStackSizeGroup
[DEBUG] Path not found: PlayerStateData.DifficultyState.Preset.DifficultyPresetType
[STDERR] Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
[STDERR] 	at nomanssave.gB.cz(Unknown Source)
[STDERR] 	at nomanssave.dm.setSelectedItem(Unknown Source)
[STDERR] 	at javax.swing.JComboBox.setSelectedItem(Unknown Source)
[STDERR] 	at javax.swing.JComboBox.setSelectedIndex(Unknown Source)
[STDERR] 	at nomanssave.dl.a(Unknown Source)
[STDERR] 	at nomanssave.Application.k(Unknown Source)
[STDERR] 	at nomanssave.Application.e(Unknown Source)
[STDERR] 	at nomanssave.Application.a(Unknown Source)
[STDERR] 	at nomanssave.D.setSelectedItem(Unknown Source)
[STDERR] 	at javax.swing.JComboBox.setSelectedItem(Unknown Source)
[STDERR] 	at javax.swing.JComboBox.setSelectedIndex(Unknown Source)
[STDERR] 	at javax.swing.plaf.basic.BasicComboPopup$Handler.mouseReleased(Unknown Source)
[STDERR] 	at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
[STDERR] 	at java.awt.Component.processMouseEvent(Unknown Source)
[STDERR] 	at javax.swing.JComponent.processMouseEvent(Unknown Source)
[STDERR] 	at javax.swing.plaf.basic.BasicComboPopup$1.processMouseEvent(Unknown Source)
[STDERR] 	at java.awt.Component.processEvent(Unknown Source)
[STDERR] 	at java.awt.Container.processEvent(Unknown Source)
[STDERR] 	at java.awt.Component.dispatchEventImpl(Unknown Source)
[STDERR] 	at java.awt.Container.dispatchEventImpl(Unknown Source)
[STDERR] 	at java.awt.Component.dispatchEvent(Unknown Source)
[STDERR] 	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
[STDERR] 	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
[STDERR] 	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
[STDERR] 	at java.awt.Container.dispatchEventImpl(Unknown Source)
[STDERR] 	at java.awt.Window.dispatchEventImpl(Unknown Source)
[STDERR] 	at java.awt.Component.dispatchEvent(Unknown Source)
[STDERR] 	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
[STDERR] 	at java.awt.EventQueue.access$500(Unknown Source)
[STDERR] 	at java.awt.EventQueue$3.run(Unknown Source)
[STDERR] 	at java.awt.EventQueue$3.run(Unknown Source)
[STDERR] 	at java.security.AccessController.doPrivileged(Native Method)
[STDERR] 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
[STDERR] 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
[STDERR] 	at java.awt.EventQueue$4.run(Unknown Source)
[STDERR] 	at java.awt.EventQueue$4.run(Unknown Source)
[STDERR] 	at java.security.AccessController.doPrivileged(Native Method)
[STDERR] 	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
[STDERR] 	at java.awt.EventQueue.dispatchEvent(Unknown Source)
[STDERR] 	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
[STDERR] 	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
[STDERR] 	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
[STDERR] 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
[STDERR] 	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
[STDERR] 	at java.awt.EventDispatchThread.run(Unknown Source)
```
<img width="862" alt="Screen Shot 2023-04-24 at 1 30 47 PM" src="https://user-images.githubusercontent.com/5948524/233907656-9086f254-832c-4405-adf6-8b8cd0c60365.png">

**Comment by Devilin-Pixy** [comment]
@goatfungus Here is a PS4 save made post-interceptor (not mine), but was fine to share.
[PS4Save.zip](https://github.com/goatfungus/NMSSaveEditor/files/11309192/PS4Save.zip)

**Comment by Devilin-Pixy** [comment]
@MHO-Gzus I am not sure, but I have no doubt it will get sorted soon. It may be useful to include a copy of your saves as well though, just to be sure.

**Comment by MHO-Gzus** [comment]
This is the saved game i was editing when the NMS editor asked to update/restart, and it reloaded with the error that blanks out the tabs. 

[savedata07.zip](https://github.com/goatfungus/NMSSaveEditor/files/11309926/savedata07.zip)

**Comment by MHO-Gzus** [comment]
Java updated! it seems that after the java update is when my issue started. maybe that will help point someone in the right direction. 
![image](https://user-images.githubusercontent.com/131669367/234076083-6be2f918-46be-4281-978c-011490ac98ad.png)

**Comment by goatfungus** [comment]
> Looking at an older save (pre-Fractals) compared to new save, it appears PS4 now uses 'mapping' as well? Never really looked into PS4 saves myself before to be honest

This is most likely the issue, so I'm going to add optional mapping to PS4 saves and see if that works.

**Comment by goatfungus** [comment]
This should now be fixed in version 1.13.4.
